src/services/API/Plio.js

```diff
@@ -48,13 +52,57 @@ export default {
     };
     return apiClient().post(process.env.VUE_APP_BACKEND_PLIOS, newPlioData);
   },
-  updatePlio(value, plioId) {
-    // creates a new draft plio
-    var plioData = {
-      value: value,
-      plio_id: plioId,
-      bucket: "plio-test-data",
-    };
-    return apiClient().post(process.env.VUE_APP_BACKEND_CREATE_PLIO, plioData);
+
+  async updatePlio(value, plioId) {
+    // handle items and questions being updated
+    value.items.forEach((item) => {
+      // pass a deepclone and not the reference
+      var cloneDeep = require("lodash.clonedeep");
+      var itemClone = cloneDeep(item);
+      ItemService.updateItem(itemClone);
+      QuestionService.updateQuestion(itemClone.details);
+    });
+
+    // prepare the payload for plio
+    var plioObject = {};
+
+    // handle (plio_title, status, created_by) being updated
+    plioObject.name = value.plio_title;
+    plioObject.status = value.status;
+    plioObject.created_by = 1; // TODO - HARDCODED FOR NOW
```

Comment:
<img width="389" alt="Screenshot 2021-04-07 at 9 04 10 PM" src="https://user-images.githubusercontent.com/13185699/113893596-ce098480-97e4-11eb-89e1-93d2665e2bfe.png">
I am converting in Editor.vue to the S3 format and you are converting from S3 format to DB format. We can directly go from JS format to DB format - and this should be done in the same place where the DB to S3 format conversion is happening in Editor.vue